test/test_ensemble_hydrostatic_free_surface_models.jl

```diff
@@ -4,6 +4,41 @@ using Oceananigans.Models.HydrostaticFreeSurfaceModels: ColumnEnsembleSize, Slic
 using Oceananigans.TurbulenceClosures: ConvectiveAdjustmentVerticalDiffusivity
 const CAVD = ConvectiveAdjustmentVerticalDiffusivity
 
+@testset "`HydrostaticFreeSurfaceModel` using a `SingleColumnGrid`" begin
+
+    Nz = 3
+    Hz = 1
+    single_column_topology = (Flat, Flat, Bounded)
+    periodic_topology = (Periodic, Periodic, Bounded)
+
+    single_column_grid = RectilinearGrid(; size=Nz, z=(-1, 0), topology = single_column_topology, halo=Hz)
+    periodic_grid = RectilinearGrid(; size=(1, 1, Nz), z=(-1, 0), topology = periodic_topology, halo=(1, 1, Hz))
+    coriolis = FPlane(f=0.2)
+    closure  = CAVD(background_κz=1.0)
+
+    Δt = 0.01
+
+    model_kwargs = (; tracers=:c, buoyancy=nothing, closure)
+    simulation_kwargs = (; Δt, stop_iteration=100)
+
+    sic_model = HydrostaticFreeSurfaceModel(; grid, coriolis=coriolises[i, j], model_kwargs...)
+    per_model = HydrostaticFreeSurfaceModel(; grid, coriolis=coriolises[i, j], model_kwargs...)
+
+    set!(sic_model, c = z -> exp(-z^2), u = 1, v = 1)
+    set!(per_model, c = z -> exp(-z^2), u = 1, v = 1)
+
+    sic_simulation = Simulation(sic_model; simulation_kwargs...)
+    per_simulation = Simulation(per_model; simulation_kwargs...)
+    run!(sic_simulation)
+    run!(per_simulation)
+    
+    @info "Testing Single column grid results..."
+    
+    @test all(sic_model.velocities.u.data[i, j, :] .≈ per_model[i, j].velocities.u.data[1, 1, :])
+    @test all(sic_model.velocities.v.data[i, j, :] .≈ per_model[i, j].velocities.v.data[1, 1, :])
+    @test all(sic_model.tracers.c.data[i, j, :]    .≈ per_model[i, j].tracers.c.data[1, 1, :])
+end
+
```

Comment:
was this sent to the wrong PR?

Reply:
yeah this was meant to be in #3741. I have moved it there